cfgov/cfgov/settings/base.py

```diff
@@ -177,7 +177,8 @@
 # Absolute path to the directory static files should be collected to.
 STATIC_ROOT = os.environ.get('DJANGO_STATIC_ROOT', '/var/www/html/static')
 
-MEDIA_ROOT = os.path.join(PROJECT_ROOT, 'f')
+MEDIA_ROOT = os.environ.get('MEDIA_ROOT', 
+                            os.path.join(PROJECT_ROOT, 'f'))
```

Comment:
> So rather than fall back on a value if the environment variable is not set, it would be better if we set the environment variables with the default values. It could be done in this file.

We can do it that way if that's the consensus, but I'm very much not sold on the benefit of *not* having sane defaults in the actual configuration if the environment doesn't happen to contain the variable.

On top of this, by defaulting to `PROJECT_ROOT/f`, this is consistent with the previous behavior.